UnityProject/Assets/Scripts/PlayGroups/P2PInteractions.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using PlayGroups.Input;
+using UI;
+using Weapons;
+
+namespace PlayGroup
+{
+	/// <summary>
+	/// Player 2 Player interactions. Also used for clicking on yourself
+	/// </summary>
+	public class P2PInteractions : InputTrigger
+	{
+		public override void Interact(GameObject originator, Vector3 position, string hand)
+		{
+			if(Input.GetKey(KeyCode.LeftControl) || Input.GetKey(KeyCode.LeftCommand)){
+				return;
+			}
+
+			if(UIManager.Hands.CurrentSlot.Item != null){
```

Comment:
This code should be telling the item to do an interaction. Right now it's just horribly hard-coded.